app/delivery/send_to_providers.py

```diff
@@ -82,6 +82,8 @@ def send_sms_to_provider(notification):
 
                     if my_phone:
                         my_phone = my_phone.decode("utf-8")
+                    # TODO REMOVE
+                    current_app.logger.info(f"IN SEND TO PROVIDERS, WHERE WE GET THE VALUE, KEY IS {key} and value is {my_phone}")
```

Comment:
Looks like this line is too long:

```suggestion
                    current_app.logger.info(
                        f"IN SEND TO PROVIDERS, WHERE WE GET THE VALUE, KEY IS {key} and value is {my_phone}"
                    )
```